cli/functionary/environment.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+import click
+import requests
+
+from .config import get_config_value, save_config_value
+
+
+def get_environment_list():
+    """
+    Helper function to get the environment list from host
+
+    Args:
+        None
+
+    Returns:
+        None
+
+    Raises:
+        ClickException if bad get request
+    """
+    token = get_config_value("token")
+    teams_url = get_config_value("host") + "/api/v1/teams"
+    header = {"Authorization": f"Token {token}"}
+    response = requests.get(teams_url, headers=header)
+
+    if response.ok:
+        data = json.loads(response.text).get("results")
+
+        env_list = []
+        for team in data:
+            for env_set in team.get("environments"):
+                env_list.append(env_set)
+        return env_list
+    else:
+        raise click.ClickException(
+            f"Failed to get environment list: {response.status_code}\n"
+            f"Response: {response.text}"
+        )
+
+
+@click.group("environment")
+@click.pass_context
+def environment_cmd(ctx):
```

Comment:
You'll want a basic docstring here so that the usage has a nice summary.  It's used to fill in the command description when you just run `functionary` on its own:

```shell
$ functionary
Usage: functionary [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  environment  # docstring summary will populate text here
  login        Login to Functionary.
  package
```